libs/wire-api/src/Wire/API/Team/Feature.hs

```diff
@@ -21,56 +19,62 @@
 -- with this program. If not, see <https://www.gnu.org/licenses/>.
 
 module Wire.API.Team.Feature
-  ( -- * LegalHoldTeamConfig
-    LegalHoldTeamConfig (..),
-    LegalHoldStatus (..),
-
-    -- * SSOTeamConfig
-    SSOTeamConfig (..),
-    SSOStatus (..),
-
-    -- * Swagger
-    modelLegalHoldTeamConfig,
-    modelSsoTeamConfig,
+  ( TeamFeatureName (..),
+    typeFeatureName,
```

Comment:
I usually kept the Swagger models separate in the export list to avoid the noise. Do you think that makes sense or should we export them next to the types they describe?

Reply:
No, your approach is fine, this is an oversight.  I make a separate PR to fix this.

Reply:
https://github.com/wireapp/wire-server/pull/1126